jetson/nav/search/searchStateMachine.cpp

```diff
@@ -116,6 +122,12 @@ NavState SearchStateMachine::executeRoverWait()
                                        mRover->roverStatus().odometry().bearing_deg );
         return NavState::TurnToTarget;
     }
+    else if( mRover->roverStatus().leftTarget().distance == -1 && 
```

Comment:
copy-pasta error? should this say rightTarget?

Reply:
No, this is fine as is

Reply:
No, at this point, the rover is trying to wait before locating the first target (which is the `leftTarget` one)